SQL/0000-00-00-schema.sql

```diff
@@ -359,7 +359,7 @@ CREATE TABLE `test_subgroups` (
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8;
 
 
-INSERT INTO test_subgroups (Subgroup_name) VALUES ('Instruments');
+INSERT INTO test_subgroups (Subgroup_name) VALUES ('Instruments'),('Imaging');
```

Comment:
@driusan I added this to the schema becaue some of our "template" instruments assume it exists...

should I add it here or remove it from the instruments ?

Reply:
I think it makes sense to include it here